src/locales/langs/de.json

```diff
@@ -144,6 +144,25 @@
       "item21Title": "Kontaktinformationen",
       "item21Text": "Wenn Sie Fragen oder Bedenken zu diesen Allgemeinen Geschäftsbedingungen haben, kontaktieren Sie uns bitte unter Welcome@zenbit.tech."
     }
+  },
+   "ChangePasswordPage": {
+      "title": "New password",
+      "passwordInput": "Password",
+      "passwordInputPlaceholder": "Min 8 characters",
+      "repeatPasswordInput": "Repeat password",
+      "emailRequired": "Email is required",
+      "emailInvalid": "Invalid email",
+      "passwordRequired": "Password is required",
+      "minChar": "Minimum 8 characters",
+      "repeatLabel": "Repeat password",
+      "unmatchPass": "Passwords do not match",
+      "agree": "By continuing, you agree to AgentWise",
+      "and": "and"
+  },
+  "RestorePasswordDonePage":{
+    "title": "Password reset was successfull",
+    "text": "Your password has been successfully reset. Click below to log in magically",
+    "buttonText": "Log in"
   },
   "VerifyEmail": {
     "title": "Bitte überprüfen Sie Ihre E-Mails!",
```

Comment:
why english text in de.json?

Reply:
We don't use german now as I know

Reply:
no, I suggest delete this file, to avoid confusion